app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/MultipleDisposalsTriageController.scala

```diff
@@ -1398,8 +1400,12 @@ class MultipleDisposalsTriageController @Inject() (
       }
     }
 
-  def isAValidCGTTaxTear(taxYearExchanged: TaxYearExchanged): Boolean =
-    !(taxYearExchanged === TaxYearExchanged.TaxYearBefore2020 || taxYearExchanged === TaxYearExchanged.DifferentTaxYears)
+  def isAValidCGTTaxTear(taxYearExchanged: TaxYearExchanged, stateYear: Option[String] = None): Boolean =
+    TaxYearExchanged.taxYearsMap.get(stateYear.getOrElse("")) match {
```

Comment:
This function returns a valid CGT taxyear that means 2020, 2021 and avoids taxyears which are before 2020.
Use some other function and avoid using `getOrElse("")` ( means using just string)

Reply:
We are not having TaxYear calculations for before 2020 years. So it will rightly redirect to error message.

Reply:
Existing `isAValidCGTTaxTear ` purpose is different and your need is different. You should write a function like `isOriginalSumissionYear(.........):Boolean`